assets/js/modules/tagmanager/components/common/FormInstructions.js

```diff
@@ -47,12 +47,19 @@ export default function FormInstructions( { isSetup } ) {
 	const hasMultipleAnalyticsPropertyIDs = useSelect( ( select ) =>
 		select( MODULES_TAGMANAGER ).hasMultipleAnalyticsPropertyIDs()
 	);
-	const analyticsPropertyID = useSelect( ( select ) =>
-		select( MODULES_ANALYTICS ).getPropertyID()
+	const analyticsModuleAvailable = useSelect( ( select ) =>
+		select( CORE_MODULES ).isModuleAvailable( 'analytics' )
```

Comment:
I think this could be selected within the select for `analyticsPropertyID` below as the only other use already relies on `analyticsModuleActive` which should be sufficient.

Reply:
The check further down for `analyticsModuleAvailable` is required in conjunction with `! analyticsModuleActive` - it's not sufficient to simply rely on `! analyticsModuleActive`.

This is because we only want to handle the "not active" case when the module _does_ exist - in this particular case, we only want to show the text containing "Activate the Google Analytics module" when the module exists to be activated, even while `analyticsModuleActive` returns `false`.